package.json

```diff
@@ -4,6 +4,9 @@
   "description": "Kin SDK for the Node runtime",
   "main": "scripts/bin/index.js",
   "types": "scripts/bin/index.d.ts",
+  "runtimeArgs": [
```

Comment:
should be removed (local change for npm link?)

Reply:
Exactly.